test/spec/services/simulationService.js

```diff
@@ -56,16 +15,20 @@ describe('SimulationService', function () {
             });
         });
 
-        it('should return the complement from droitsDescription with the given droits', function() {
-            var actual = service.complement({ acs: { montant: 12 } });
-
-            expect(actual.acs).toBe(undefined);
-            expect(actual.apl.id).toBe('apl');
+        it('should sort eligible and non eligible droits', function() {
+            var openfiscaResult = { acs: 10 };
+            var droits = service.sortDroits(openfiscaResult);
+            expect(droits.droitsEligibles).toEqual({ acs: { id: 'acs', montant: 10 } });
+            expect(droits.droitsNonEligibles).toEqual({ apl: { id: 'apl' } });
+            expect(droits.droitsInjectees).toEqual({});
```

Comment:
Whenever possible, have each `expect` in its own `it` and group them within a `describe`, if necessary with a `before`. That makes for shorter, more specific tests that are more precise when they fail and better support evolution of the codebase.